examples/operator/operator.yaml

```diff
@@ -6085,10 +6085,6 @@ spec:
         - --enable-leader-election
         command:
         - /contour-operator
-        - --contour-image
-        - docker.io/projectcontour/contour:main
```

Comment:
I *think* this one (`--contour-image`) is still needed, because this is what gets updated to the version-specific tag when a release is cut.

Reply:
Dropping the Envoy ones is helpful, thanks!